config/routes.rb

```diff
@@ -10,11 +10,13 @@
   
   # name to prevent resource: user and devise routes from overlapping
   devise_for :users, :path => 'member'
-  
   resources :users do 
     resources :application
     resources :health_form
+    resources :questionnaire
```

Comment:
We don't need resources for questionnaire because we are not going to use all the routes. Maybe change it to something like 'get "questionairre", to: "QuestionnaireController#Index" , as: "questionnaire_index" '. you could also change the word index to just questionairre or something more meaningful/simplier, up to you as we will not be using resources and all the routes it provides.